persistence/src/commonMain/sqldelight/com/quran/shared/persistence/bookmark_tags.sq

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE bookmark_tags (
+  local_id INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
+  bookmark_local_id TEXT NOT NULL,
+  bookmark_type TEXT NOT NULL, -- 'PAGE' or 'AYAH'
```

Comment:
wdyt of this - make it a number or split this into two tables or?

Reply:
It looks good to me the way it's right now